bin/sdTest.sh

```diff
@@ -64,6 +64,7 @@ function echo_configuration {
     echo "PHP 7.0: ${SHOPWARE_VERSION}870"
     echo "PHP 5.6: ${SHOPWARE_VERSION}856"
     echo "MySQL:   ${SHOPWARE_VERSION}331"
+    echo "Mailhog: 80${SHOPWARE_VERSION}"
```

Comment:
hat das einen Grund warum du das hier anders gemacht hast?

Reply:
what exactly do you mean?

Reply:
On the first lines it is used like variable -> port and you wrote port -> variable

Reply:
The idea behind this is to have the `mailhog port` closer to the `default` port of `mailhog`.

The other Ports are imho, something different, because there is also the `PHP`-version coded in there. so basically it is something like `${SHOPWARE_VERSION}8{$PHP_VERSION}`